framework/routing/__init__.py

```diff
@@ -392,12 +413,15 @@ def __init__(self, template_name, renderer=None, error_renderer=None,
                      to add to data from view function
         :param detect_render_nested: Auto-detect renderers for nested
             templates?
+        :param safe: Boolean: turn on markup-safe escaping?
```

Comment:
In Mako, the "html safe" filter is applied when the data is known to be safe, not when you want to make it safe. I think using the word "safe" to mean the opposite here could be confusing. Perhaps flipping the boolean and renaming it "unsafe" would work?